docs/dev/15.core/forms.mdx

```diff
@@ -227,10 +227,10 @@ Markdown is currently only rendered in the web-admin
 
 #### Example
 
- ``toml
+```toml
 [section.my_option_id]
-type = "markdown"
-ask = "Text **rendered** in markdown.\nAnd this text is on a new line."
+type = "display_text"
```

Comment:
Hmmm that's odd considering this section is supposed to be for the "markdown" type x_X ?